### ISSUE TITLE
Access dynamically created Angular components

### ISSUE THREAD

**Issue by jayhaluska** [EmbeddedEnterprises/ng6-golden-layout#42]
I want to access the Angular component instances which GoldenLayout creates for me.

I have a parent component with a GL root element. I initialize the layout empty and create a component dynamically:
``` js
this.cmp.createNewComponent({
    componentName: 'myComponent',
    type: 'component',
    componentState: { ... }
})
```

The created component offers events to which I want to subscribe.
I thought that there must be some way of getting a `ComponentRef`, the objects which `ComponentFactory`s create, so that I can call:
``` js
referencoToAngularComponent.instance.outputEvent.subscribe(myFunction);
```

How do we access the references to Angular components?

Angular version: 8

Thanks in advance for any help and also for this angular integration. The work is much appreciated.

**Comment by martin31821** [comment]
How would you think an API to achieve such functionality should look like?

**Comment by jayhaluska** [comment]
The most obvious way of achieving this would be if `createNewComponent` would return a `ComponentRef` instead of `void`.

But what I do not understand is that `createNewComponent` does not seem to create an Angular component. It just calls `createContentItem` of the underlying GoldenLayout library. How is the Angular component created?

The only place where an element with type `ComponentRef` is created is in the initialize method.

Sorry if I missed something. Just had a glance at the code.

**Comment by pmalde** [comment]
Hi just following on from this. Is there any way I can get hold of the component reference for an Angular component?

**Comment by martin31821** [comment]
fixed in 251f301

**Comment by pmalde** [comment]
Awesome. Thanks Martin. Great contribution.
